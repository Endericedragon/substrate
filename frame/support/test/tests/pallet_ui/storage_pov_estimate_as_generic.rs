#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::*;

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(_);

	#[pallet::error]
	pub enum Error<T> {}

	#[pallet::storage]
	type Foo<T: Config> = StorageValue<Value = u8, ProofSize = frame_support::storage::MeasuredProofSize>;
}

fn main() {
}
