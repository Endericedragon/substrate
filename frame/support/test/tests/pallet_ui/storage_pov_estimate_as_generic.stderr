error: Invalid pallet::storage, Unexpected generic `ProofSize` for `StorageValue`. `StorageValue` expect generics `Value`, and optional generics `QueryKind`, `OnEmpty`.
  --> tests/pallet_ui/storage_pov_estimate_as_generic.rs:15:49
   |
15 |     type Foo<T: Config> = StorageValue<Value = u8, ProofSize = frame_support::storage::MeasuredProofSize>;
   |                                                    ^^^^^^^^^
